src/extension.ts

```diff
@@ -36,60 +36,62 @@ let unsupportedDevVersion = false;
-
-
-
-    // BX CS commands *************************************
-    registerPromptingCommand(context, 'extension.bx.cs.cluster-create', {cmd: 'bx', args: ['cs', 'cluster-create']}, outputChannel, [new PromptInput('Specify a cluster name', '--name')]);
-    registerPromptingCommand(context, 'extension.bx.cs.cluster-get', {cmd: 'bx', args: ['cs', 'cluster-get']}, outputChannel, [new PromptInput('Specify a cluster name or id')]);
-    registerPromptingCommand(context, 'extension.bx.cs.cluster-rm', {cmd: 'bx', args: ['cs', 'cluster-rm']}, outputChannel, [new PromptInput('Specify a cluster name or id')], ['-f']);
-    registerCommand(context, 'extension.bx.cs.clusters', {cmd: 'bx', args: ['cs', 'clusters']}, outputChannel);
-    registerCommand(context, 'extension.bx.cs.init', {cmd: 'bx', args: ['cs', 'init']}, outputChannel);
-    registerPromptingCommand(context, 'extension.bx.cs.worker-get', {cmd: 'bx', args: ['cs', 'worker-get']}, outputChannel, [new PromptInput('Specify worker id')] );
-    registerPromptingCommand(context, 'extension.bx.cs.worker-add', {cmd: 'bx', args: ['cs', 'worker-add']}, outputChannel, [new PromptInput('Specify cluster name or id')], ['1'] );
-    registerPromptingCommand(context, 'extension.bx.cs.worker-reboot', {cmd: 'bx', args: ['cs', 'worker-reboot']}, outputChannel, [new PromptInput('Specify a cluster name or id'), new PromptInput('Specify a worker id')], ['-f'] );
-    registerPromptingCommand(context, 'extension.bx.cs.worker-reload', {cmd: 'bx', args: ['cs', 'worker-reload']}, outputChannel, [new PromptInput('Specify a cluster name or id'), new PromptInput('Specify a worker id')], ['-f'] );
-    registerPromptingCommand(context, 'extension.bx.cs.worker-rm', {cmd: 'bx', args: ['cs', 'worker-rm']}, outputChannel, [new PromptInput('Specify a cluster name or id'), new PromptInput('Specify a worker id')], ['-f'] );
-    registerPromptingCommand(context, 'extension.bx.cs.workers', {cmd: 'bx', args: ['cs', 'workers']}, outputChannel, [new PromptInput('Specify a cluster name or id')] );
+    console.log('Congratulations, your extension "com-ibm-cloud" is now active!');
+
+    LoginManager.registerCommand(context, 'extension.ibmcloud.login');
+    LoginManager.registerCommand(context, 'extension.ibmcloud.login.sso');
+    registerCommand(context, 'extension.ibmcloud.logout', {cmd: 'ibmcloud', args: ['logout']}, outputChannel);
+    registerCommand(context, 'extension.ibmcloud.cli-update', {cmd: 'ibmcloud', args: ['plugin', 'update', '-r', 'IBMCloud']}, outputChannel);
+
+
+    // IBMCloud DEV commands *************************************
+    registerCommand(context, 'extension.ibmcloud.dev.list', {cmd: 'ibmcloud', args: ['dev', 'list', '--caller-vscode']}, outputChannel, true);
+    registerCommand(context, 'extension.ibmcloud.dev.create', {cmd: 'ibmcloud', args: ['dev', 'create', '--caller-vscode']}, outputChannel, true, SystemCommand, true);
```

Comment:
Prior version did not have `create`, but instructed the user to do this step outside of the extension.

Reply:
Right. Let me remove the new command here. I think we decided to hold off on adding new commands for the new release.